### PR TITLE
To find the Probability of getting at least K heads in N tosses of Coins in Java

### LINKED ISSUE #331: Concept of probability and binomial  theorem
# Description
To find the Probability of getting at least K heads in N tosses of Coins

# Important Checklist

[. ] I have read the contribution guidelines before making this issue.
[. ] I have verified this issue program is not already implemented
[ .] I have not pushed this code anywhere else (even my repo)

# Languages and Assignees:
-   [x] Java: @aastha01chauhan

### PR DESCRIPTION
**Pull request by aastha0326**
To Find The Probability of getting at least K heads in N tosses of Coins in JAVA

# Description
I have endeavoured to provide the most optimised solution by myself. This problem revolves around the concept of probability and binomial theorem.

Fixes #331 Probability of getting at least K heads in N tosses of Coins

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

# Checklist:

-   [ ] I have read the contribution guidelines before making this PR.
-   [ ] My code follows the style guidelines of this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented in my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [ ] My changes generates no new warnings
-   [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
- The file name is incorrect without any extension. 
- Java folder readme has not been updated
- Pull request template not filled correctly

File: Java/Probability of getting at least K heads in N tosses of CoinsProbability of getting at least K heads in N tosses of Coins [hunk 1]
```diff
@@ -0,0 +1,54 @@
+// JAVA Code for Probability of getting
```
Comment (YashSaxena9) [inline Java/Probability of getting at least K heads in N tosses of CoinsProbability of getting at least K heads in N tosses of Coins:1]: file extension missing, and code format is not good

**Review by YashSaxena9 — CHANGES_REQUESTED** [review]

**Comment by aastha0326** [comment]
I'll just correct it.

**Comment by kaustubhgupta** [comment]
Failing to do changes will reject this PR, just a heads up

**Pull request closed without merging**